03_launch_mgmt_cluster.sh

```diff
@@ -154,7 +154,7 @@ function launch_baremetal_operator() {
 }
 
 function launch_kind() {
-  reg_ip="$(docker inspect -f '{{.NetworkSettings.IPAddress}}' registry)"
+  reg_ip="$(sudo docker inspect -f '{{.NetworkSettings.IPAddress}}' registry)"
```

Comment:
maybe we should make a deeper fix, as this will only result in hitting further problems. We probably should add the current user to the docker group and use `sudo su -l -c "command to run" "$USER"` wherever we are running kind commands to make sure that our user can use the docker socket.

Reply:
Thanks. Working on it .

Reply:
Done